src/pyflex/window_selector.py

```diff
@@ -745,18 +824,18 @@ def reject_based_on_criteria(win):
             dlnA_max = self.config.dlna_reference + \
                 self.config.dlna_acceptance_level[win.center]
 
-            if not (tshift_min < win.cc_shift *
-                    self.observed.stats.delta < tshift_max):
-                logger.debug("Window rejected due to time shift: %f" %
-                             win.cc_shift)
+            if not (tshift_min <= win.cc_shift *
+                    self.observed.stats.delta <= tshift_max):
+                logger.debug("Window rejected due to time shift:    %6.2f %6.1f %6.1f" %
+                             (win.cc_shift, win.relative_starttime, win.relative_endtime))
```

Comment:
Can you make these a bit more descriptive? Otherwise it will be hard to make sense of them in a couple of months. Something along the lines of

```python
logger.debug("Window [%.1f - %.1f seconds] rejected due to time shift: %.1f" % ...)
```